pkgs/top-level/python-packages.nix

```diff
@@ -14994,4 +14994,94 @@ let
     };
   };
 
+  
+  markdown2 = buildPythonPackage rec {
+    name = "markdown2-${version}";
+    version = "2.3.0";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/m/markdown2/${name}.zip";
+      sha256 = "073zyx3caqa9zlzxa82k9k2nhhn8c5imqpgp5nwqnh0fgaj9pqn8";
+    };
+    propagatedBuildInputs = with self; [];
+    meta = {
```

Comment:
your metas' rhs scope doesn't contain licenses or maintainers. you need to bring them in either by being explicit or adding `with stdenv.lib;` to the attrset declaration.